core/jvm/src/main/scala/zio/blocking/package.scala

```diff
@@ -168,7 +168,7 @@ package object blocking {
       Executor.fromThreadPoolExecutor(_ => Int.MaxValue) {
         val corePoolSize  = 0
         val maxPoolSize   = Int.MaxValue
-        val keepAliveTime = 1000L
+        val keepAliveTime = 60000L
```

Comment:
I think maybe we should change `maxPoolSize` to something like 1000.

Reply:
Done.